### ISSUE TITLE
Issue with "Django Tutorial Part 10: Testing a Django web appl…": actual value first to keep the pattern

### ISSUE THREAD

**Issue by andrehora** [mdn/content#4886]
MDN URL: https://developer.mozilla.org/en-US/docs/Learn/Server-side/Django/Testing

#### What information was incorrect, unhelpful, or incomplete?

This is a minor suggestion.

Just to keep the pattern with other assertions on the page, it would be better to write the actual value first:
```python
self.assertEqual(str(author), expected_object_name)
```

Instead of the expected value first (as currently presented in two occurrences):
```python
self.assertEqual(expected_object_name, str(author))
```

Other assertions on the page already follow this pattern (ie, actual & expected):

```python
self.assertEqual(field_label, 'first name')
self.assertEqual(field_label, 'died')
self.assertEqual(max_length, 100)
self.assertEqual(author.get_absolute_url(), '/catalog/author/1')
...
```

#### Specific section or headline?

LocalLibrary tests>>Models

#### What did you expect to see?

Replace the 2 occurrences of:

```python
self.assertEqual(expected_object_name, str(author))
```

by:

```python
self.assertEqual(str(author), expected_object_name)
```

#### Did you test this? If so, how?

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/learn/server-side/django/testing`
* MDN URL: https://developer.mozilla.org/en-US/docs/Learn/Server-side/Django/Testing
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/learn/server-side/django/testing/index.html
* Last commit: https://github.com/mdn/content/commit/3af43c1dcfb9be19c73cf3fdd1178db9c74aa064
* Document last modified: 2021-04-27T01:03:07.000Z

</details>

**Comment by andrehora** [comment]
The same applies to the following code in method `test_only_borrowed_books_in_list`:

```python
self.assertEqual('o', bookitem.status)
```

This could actually be:
```python
self.assertEqual(bookitem.status, 'o')
```

**Comment by hamishwillee** [comment]
Thanks @andrehora . I have fixed as part of #4896.

FWIW This kind of inconsistency is annoying but not particularly damaging to the example - so normally it would be very low priority. Please free to tidy this kind of thing by creating a PR, noting that we have to also keep the demo version of the tutorial in sync: https://github.com/mdn/django-locallibrary-tutorial

**Comment by andrehora** [comment]
Sure, thanks @hamishwillee for the fast response!
